spring-kafka/src/main/java/org/springframework/kafka/support/micrometer/KafkaRecordReceiverContext.java

```diff
@@ -60,6 +89,14 @@ public String getListenerId() {
 		return this.listenerId;
 	}
 
+	public String getGroupId() {
+		return this.groupId;
+	}
+
+	public String getClientId() {
+		return this.clientId;
```

Comment:
Apparently both these new props can be `null`, so let's add `@Nullable` on their methods!